src/main/java/io/spokestack/spokestack/profile/AndroidASR.java

```diff
@@ -0,0 +1,23 @@
+package io.spokestack.spokestack.profile;
```

Comment:
this profile is anemic and doesn't pull its weight - the profiles should be built around ensembles of pipeline components (google-push-to-talk, etc.) and not just encapsulate a string value